R/pkg/R/stats.R

```diff
@@ -134,9 +129,7 @@ setMethod("freqItems", signature(x = "SparkDataFrame", cols = "character"),
             collect(dataFrame(sct))
           })
 
-#' approxQuantile
-#'
-#' Calculates the approximate quantiles of a numerical column of a SparkDataFrame.
+#' approxQuantile - Calculates the approximate quantiles of a numerical column of a SparkDataFrame.
```

Comment:
@felixcheung . I found that `@description` and `@details` works for me.
```
#' @description
#' approxQuantile - Calculates the approximate quantiles of a numerical column of a SparkDataFrame.
#'
#' @details
#' The result o~~
```

If we keep the description, please try the above pair.